jtr-isolate.yml

```diff
@@ -0,0 +1,30 @@
+version: 1
+
+base-url: /test/
+
+runs:
+  module:
+    - basic
+    - ajax
+    - animation
+    - attributes
+    - callbacks
+    - core
+    - css
+    - data
+    - deferred
+    - deprecated
+    - dimensions
+    - effects
+    - event
+    - manipulation
+    - offset
+    - queue
+    - selector
+    - serialize
+    - support
+    - traversing
+    - tween
```

Comment:
Is this making each module run individual?

Reply:
Yes, same as now. Each module runs on its own page, but the browser instance is shared (so it doesn't have to be restarted for each test).